File: C/Estrutura repetitiva/media.c
// Biblioteca
#include <stdio.h>

int main()
{

    // Variável
    int soma, contador, idade;
    double media;
    soma = 0;
    contador = 0;

    // Entrada de dados
    printf("Digite as idades: ");
    scanf("%d", &idade);

    // Estrutura condicional
    if (idade < 0) {
        printf("Impossivel calcular uma idade negativa!");
    }

    else {
        // Estrutura repetitiva
        while (idade >= 0) {
            soma = soma + idade;
            contador = contador + 1;
            scanf("%d", &idade);
        }

        // Saída de dados
        media = (double) soma / contador;
        printf("A media das idades das %d pessoas e de %.2lf", contador, media);
    }

    return 0;
}

File: Python/Arranjos vetoriais/media.py
# Entrada de dados
n = int(input("Quantos elementos vai ter o vetor? "))

# Criação do vetor
vet: [int] = [0 for i in range(n)]

# Variável
soma = 0

# Estrutura repetitiva para ler os dados do vetor
for i in range(0, n):
    vet[i] = float(input("Digite um número: "))
    soma += vet[i]

# Saída dos valores
# Condicional para verificar se existe algum número par
if vet[i] %2 == 0:
    media = soma / n
    print(f"\nA média dos pares é {media:.1f}")
else:
    print("Nenhum número par foi encontrado!")